lib/Facebook.js

```diff
@@ -407,17 +409,18 @@ function Facebookbot(configuration) {
                 if (cb) { cb(null, facebook_botkit.webserver); }
             });
 
-
-        request.post('https://' + api_host + '/me/subscribed_apps?access_token=' + configuration.access_token,
-            function(err, res, body) {
-                if (err) {
-                    facebook_botkit.log('Could not subscribe to page messages');
-                } else {
-                    facebook_botkit.debug('Successfully subscribed to Facebook events:', body);
-                    facebook_botkit.startTicking();
-                }
-            });
-
+        var access_tokens = (typeof (configuration.access_token) === 'string') ? [configuration.access_token] : configuration.access_token;
+        Object.keys(access_tokens).forEach(function(page) {
```

Comment:
correct. i agree that it is best NOT to use Object.keys on an array. 

on the other hand, the interesting fact is that even when you have an array, Object.keys also works and extract only the defined indexes. (i.e. var a = []; a[123]="abc"; b[456]="def"; Object.keys(a); //this will return ["123","456"])